.github/ALLOWLIST

```diff
@@ -24,3 +24,4 @@ internal/common
 ## DEPRECATED components
 
 ## UNMAINTAINED components
+exporter/alibabacloudlogserviceexporter/
```

Comment:
This should go on a separate PR, right?

Reply:
it probably got added during the regeneration of the CODEOWNERS file.

Reply:
fixed with a rebase now, thanks!